internal/log/log_test.go

```diff
@@ -64,3 +68,23 @@ func TestPrepareLogFile_ReturnsErrIfParentDirIsReadOnly(t *testing.T) {
 	assert.Nil(t, file)
 	assert.Error(t, err)
 }
+
+func TestWithFields(t *testing.T) {
```

Comment:
If you have changed it to not include the fields for non-json logging, could you please add a testcase that asserts that as well please? Like the opposite of this one.

Reply:
it's in zap_test.go

Reply:
Ah! missed that sorry